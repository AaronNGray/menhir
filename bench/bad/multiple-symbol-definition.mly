/* This grammar is accepted by ocamlyacc, but rejected by Menhir,
   as there are two definitions of the symbol [a], and we view
   this as a potential mistake. We allow splitting the definition
   of %public symbols only. */
%token A B C
%start a
%type<unit> a
%%
a: A B C {}
a: B     {}
