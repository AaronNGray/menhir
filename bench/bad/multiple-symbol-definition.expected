File "multiple-symbol-definition.mly", line 9, characters 0-1:
File "multiple-symbol-definition.mly", line 10, characters 0-1:
Error: the nonterminal symbol a is multiply defined.
